frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -403,17 +446,12 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
   return (
     <div className="modal-content">
       <ModalTitle>
-        {isEditing ? t('kubevirt-plugin~Edit') : t('kubevirt-plugin~Add')}{' '}
-        {t('kubevirt-plugin~{{type}}', { type: type.toString() })}
+        <Trans t={t} ns="kubevirt-plugin" values={{ type: type.toString() }}>
```

Comment:
Trans is used when you have a text with some component, like icon or link,
here you should use `t`, think of the translator and what he/she see when they get to translate this string

Reply:
applied